File: Conversor de moedas/Model/CurrencyLiveResult.swift
//
//  CurrencyLiveResult.swift
//  Conversor de moedas
//
//  Created by Matheus Duraes on 23/12/20.
//

import Foundation

struct CurrencyLiveResult: Codable {
    let success: Bool
    let terms, privacy: String
    let timestamp: Int
    let source: String
    let quotes: [String: Double]
}

File: app/src/main/java/com/matheus/conversordemoedas/model/CurrencyLiveResult.kt
package com.matheus.conversordemoedas.model

import com.google.gson.annotations.SerializedName

class CurrencyLiveResult (
    @SerializedName("success") val success : Boolean,
    @SerializedName("terms") val terms : String,
    @SerializedName("privacy") val privacy : String,
    @SerializedName("timestamp") val timestamp : Int,
    @SerializedName("source") val source : String,
    @SerializedName("quotes") val quotes : Quotes
)